.github/workflows/tests.yml

```diff
@@ -72,3 +72,19 @@ jobs:
         run: .github/workflows/scripts/run_tests_go.sh
       - name: Go linters
         run: .github/workflows/scripts/lint_go.sh
+
+  Rust:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-18.04]
```

Comment:
Should this be ubuntu-latest so that it gets updated down the track, or is this intentionally fixed to 18.04?

Reply:
Definitely wasn't intentional. I've updated it to use `ubuntu-latest`.